src/components/Users/UserSection.jsx

```diff
@@ -17,21 +17,19 @@ const UserSection = ({ loadingUsers, loadingActive }) => {
 
   localStorage.setItem('restorePath', location.pathname);
 
-  return (
-    <>
-      <ManageUsers />
-      {loadingUsers ? (
-        <UsersList loadingActive={loadingActive} />
-      ) : (
-        <section className="panel">
-          <strong>Users List</strong>
-          <br />
-          <br />
-          <div style={{ marginBottom: '20px', textAlign: 'center' }}>Loading users list...</div>
-        </section>
-      )}
-    </>
-  );
+  return  <>
+        <ManageUsers />
+
+        {loadingUsers ? (
+          <section className="panel">
```

Comment:
The logic here was inverted. We should display the loading screen when `loadingUsers` is true